app/admin/movies.rb

```diff
@@ -1,3 +1,13 @@
 ActiveAdmin.register Movie do
-  permit_params :genre, :title, :url
+  permit_params :genre, :title, :url, :text_id
+  
+  form do |f|
+    f.inputs do
+      input :genre
+      input :title
+      input :url
+      input :text_id
```

Comment:
movie と text に関連付けを入れておけば，次の書き方で選択式になります。

```suggestion
      input :text
```